test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/VideoViewTest.cs

```diff
@@ -0,0 +1,135 @@
+﻿
+using global::System;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Samples
+{
+    using tlog = Tizen.Log;
+
+    public class VideoViewTest : IExample
+    {
+        public class myPlayer : Tizen.Multimedia.Player
+        {
+            public myPlayer() : base()
+            {
+                //Initialize();
+            }
+
+            public myPlayer(IntPtr p) : base(p, null)
+            {
+                //Initialize();
+            }
+        }
+
+        Window win;
+        myPlayer player;
+        string resourcePath;
+        const string tag = "NUITEST";
+        public void Activate()
+        {
+            win = Window.Instance;
```

Comment:
do not use Window.Instance. I will be deprecated.

Reply:
Hello, I changed as NUIApplication.GetDefaultWindow(). thank you.